sdk/attestation/azure-security-attestation/src/main/java/com/azure/security/attestation/AttestationClient.java

```diff
@@ -10,14 +10,17 @@
 import com.azure.core.http.rest.Response;
 import com.azure.core.util.Context;
 import com.azure.security.attestation.implementation.AttestationsImpl;
+import com.azure.security.attestation.implementation.models.TpmAttestationRequest;
+import com.azure.security.attestation.implementation.models.TpmAttestationResponse;
 import com.azure.security.attestation.models.AttestOpenEnclaveRequest;
 import com.azure.security.attestation.models.AttestSgxEnclaveRequest;
 import com.azure.security.attestation.models.AttestationResponse;
 import com.azure.security.attestation.models.CloudErrorException;
-import com.azure.security.attestation.models.TpmAttestationRequest;
-import com.azure.security.attestation.models.TpmAttestationResponse;
 
-/** Initializes a new instance of the synchronous AttestationClient type. */
+import java.nio.charset.StandardCharsets;
+import java.util.Objects;
+
+/** Initializes a new instance of the synchronous AzureAttestationRestClient type. */
 @ServiceClient(builder = AttestationClientBuilder.class)
 public final class AttestationClient {
```

Comment:
We try to make sync client(s) completely rely on async client and not add code directly to it. Example - https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/TextAnalyticsClient.java

Reply:
100% agreed, and that will be my intention when I finally get around to writing the actual convenience layer.

But this pull request is not the convenience layer. This pull request simply splits the generated code out and creates the skeleton of a convenience layer that is compatible with the old layer.

Reply:
In that case, I will approve this pull request and get out of your way :)